example/src/components/OperationsSection/OperationsSection.js

```diff
@@ -4,6 +4,7 @@ import 'carbon-motion/dist/index.css'
 
 const OperationsSection = () => {
   const [bookmarkAnimating, setBookmarkAnimating] = useState(false)
+  const [bookmarkAddAnimating, setBookmarkAddAnimating] = useState(false)
```

Comment:
Definitely not a blocker for merging, but for code organization purposes we have been keeping each section in alphabetical order - one of us can change it next time we are working on these same files.

Reply:
Never mind! Let's leave it as-is.  Looking at the Carbon icon guidelines, I see those are in the same order you have here.